tests/test_project.py

```diff
@@ -2969,5 +2972,74 @@ def clean_up_after_add(project, project_name):
         project.delete_project(project_name)
 
 
+def helper_test_export_project(project):
+    """Test method for export project"""
+    try:
+        result = project.export_project(
+            project_name="",
+            export_design_files=True,
+            export_result_files=True,
+            export_archive_results=True,
+            export_user_files=True,
+            export_log_files=True,
+            export_system_data=True,
+            export_file_dir="",
+            export_file_name="",
+            overwrite_existing_file=True,
+        )
+    except SherlockExportProjectException as e:
+        assert str(e) == "Export Project error : Project name is invalid"
+
+    try:
+        result = project.export_project(
+            project_name="Tutorial Project",
+            export_design_files=True,
+            export_result_files=True,
+            export_archive_results=True,
+            export_user_files=True,
+            export_log_files=True,
+            export_system_data=True,
+            export_file_dir="",
+            export_file_name="",
+            overwrite_existing_file=True,
+        )
+    except SherlockExportProjectException as e:
+        assert str(e) == "Export Project error : Export directory is invalid"
+
+    try:
+        result = project.export_project(
+            project_name="Tutorial Project",
+            export_design_files=True,
+            export_result_files=True,
+            export_archive_results=True,
+            export_user_files=True,
+            export_log_files=True,
+            export_system_data=True,
+            export_file_dir="/Test/Dir",
+            export_file_name="",
+            overwrite_existing_file=True,
+        )
+    except SherlockExportProjectException as e:
+        assert str(e) == "Export Project error : Export file name is invalid"
+
+    if project._is_connection_up():
+        try:
+            result = project.export_project(
+                project_name="",
+                export_design_files=True,
+                export_result_files=True,
+                export_archive_results=True,
+                export_user_files=True,
+                export_log_files=True,
+                export_system_data=True,
+                export_file_dir="/Test/Dir",
+                export_file_name="ExportedProject",
+                overwrite_existing_file=True,
+            )
+            pytest.fail("No exception raised when using an invalid parameter")
+        except Exception as e:
+            assert type(e) == SherlockExportProjectException
+
+
```

Comment:
I think a happy path test could be added. The tricky part is deleting the dir when the test passes.